shared/charts/backup-exporter/templates/configmap.yaml

```diff
@@ -28,7 +28,7 @@ data:
         gcloud -q compute images delete "image-disk-${snapshot}"
       fi
       gcloud -q compute images create "image-disk-${snapshot}" --source-snapshot "${snapshot}"
-      gcloud -q --verbosity=debug compute images export --log-location "${LOG_BUCKET}" --destination-uri "gs://${INTERMEDIATE_BUCKET}/${TIMESTAMP}-${snapshot}.tar.gz" --image "image-disk-${snapshot}"
+      gcloud -q compute images export --log-location "gs://${LOG_BUCKET}/logs" --destination-uri "gs://${INTERMEDIATE_BUCKET}/${TIMESTAMP}-${snapshot}.tar.gz" --image "image-disk-${snapshot}" --network "network" --subnet "nodes" --zone "us-central1-a"
```

Comment:
I'm a bit more suspicious of the hardcoded region. Perhaps this doesn't matter (though it seems like we could incur more data transfer costs than necessary if we do it "wrong") or is more complex than it looks (e.g. I'm not sure I understand the comment about "The Daisy bucket can use a different name depending on the zone where the Cloudbuild runs") but could we use TF_VAR_infra_region or something here instead?

Reply:
The GCS "Daisy bucket" is something that the `gcloud compute images export` command creates automatically, among other things. In my dev environment where I had the default network, the name of the bucket was "[project_id]-daisy-bkt", when I set the network and thus the region in stg and prd, the `gcloud compute ...` tried to use "[project_id]-daisy-bkt-us" instead. Why? I don't know, I saw the error in the logs of the backup-exporter cronjob and changed the Terraform code to avoid it.

I realize of this "actions" performed by the `gcloud compute images export` command in the errors that it throws, because I was not able to find neither the code or the documentation about what this command does in the background. So the only way to use this command by a service account in our cloud is to run several times and try to recreate the same actions that it tries to do using Terraform code. This is why the implementation of the "backup exporter" it's being so frustrating and annoying. Every time that something changes, the command tries to do a new action that is not able to do. That happened:
- When the command was fired from a container with a SA instead of a normal terminal with owner permissions.
- When the destination bucket is not in the same project of the images
- When the default network was not available in the project where it runs
- When the destination/log/intermediate buckets do not have the proper permissions to store, list and delete files.
- etc

Links related:
[Documentation about the command](https://cloud.google.com/sdk/gcloud/reference/compute/images/export)
[Script that the VM created by cloud build executes](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy_workflows/export)
[Source code of the gce_export command](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools/gce_export/main.go)
[Sample of log of the daisy export](https://00e9e64bac5679d732287da770a4581ad5acdcf59d14dcdf92-apidata.googleusercontent.com/download/storage/v1/b/gpii-gcp-stg-daisy-bkt-us/o/daisy-image-export-20190806-11:14:08-dm1sh%2Flogs%2Fdaisy.log?qk=AD5uMEuYlp7UQ7DgP8G95AjxWtCwZviqn1SFhIcTuY15TXgeRaNfdruv3dKv_ArrlgxmDLzk3tjyh-eJz1QmE5EBvJtMxBskrb9iz1rggQl90KvPl5HP5GJDW6i5x9PXOMlxhx5uu-7PkQZLRzd5cF2uZPaRkj59WAu7XDLNs_1POpp5imDQ8OaIDY3nqaUwsVshBVkJeS3T8-sE5ud7Y8qiLtdDU-sCCGQXqv6SEMMc_1koLzopfupFZxH4lFVm5dXDRrkIistuxVGMb_dOvaLcVF6cfnJSSdn_HtqYqlNJMh7dhEXDAFQUmB71YOFzbLoauySwag9-cxGxtJVME-w9IqH3d2aFc3P9mkv5ObuXsURLkl3TpIZ14p1ID4f9qyNQ9VXpU9iFDUtbdraUvemuGwdGNfYSo-SgntL7FpG-BdosEA4YqHUD5Ts8dhYKEWcHittb_Ih-0RGHmA_LQG6XFrx2oY8V8nKQ1mbi0zYoZ-TWzAPjGxgkeYqrP3MBAzD1RTsVm4SQdq1Em6nVnIWZ65E-xBOjDNsOq_7MwVu1_yni-QadXc-cbSFPDAyYq0FCmekR4t5CRDgJUanWRHvY0QxdgkaTOWrAOnqezjoRZIRo-9OWd0-mE6KGhPAlMpbsIHOvi1X2Hg9dK0I4nzkPfIkZgiPI5xKH6Gm7QZTZOSm-x1pJUfhPWqEugrIZIHQ3J8CPtvke-9rEPMDlzfifnRytLZtTEJK6FlxpR0F8QVfirrvpjSRycgYfhAaJ-eiyPANYGHDL1LYi2q_EC1GbWVNmR9tkrMDus5E1jRtVfhbKfOckgGUekUKFWeOoZFWuUx9mQ9cdCUzh4g0BB27ACTo_f5wu4A)